### PR TITLE
Transfered ExecFlowDemo example to OntoTrans2ndWorkshop

### PR DESCRIPTION
**Pull request by francescalb**
to make it public

### CODE REVIEW

File: OntoTrans2ndWorkshop/README.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+$ cd /tmp
+$ ln -s /path/to/OntoTrans2ndWorkshop
+```
+
+The minimal requirement to run this example is:
+- [ExecFlow](https://github.com/H2020-OpenModel/ExecFlow)
+  (Which requires AiiDA-core, oteapi-core, oteapi-dlite, DLite-Python, Tripper, AiiDA-shell)
+and a simlified lammpsparser.
+
+This can all be installed by running `pip install .` in the current folder
```
Comment (jesper-friis) [inline OntoTrans2ndWorkshop/README.md:21]: Please also add a link to how to install and setup AiiDA: https://aiida.readthedocs.io/projects/aiida-core/en/latest/intro/get_started.html

Maybe with a quick summary, like
- install postgresql and rabbitmq (ex: `sudo apt install git postgresql postgresql-server-dev-all postgresql-client rabbitmq-server`)
- create virtual environment (ex: `python -m venv ~/envs/aiida && source ~/envs/aiida/bin/activate`) 
- install Python packages (ex: `pip install .`)
- setup profile (ex: `verdi quicksetup`) 
- start verdi deamon (ex: `verdi daemon start`)

File: OntoTrans2ndWorkshop/demo_figures/SimulationParametersExcel.png
Comment (jesper-friis) [inline OntoTrans2ndWorkshop/demo_figures/SimulationParametersExcel.png:1]: Are these figures used anywhere?

Is the plan to add them to demo.ipynb?

File: OntoTrans2ndWorkshop/README.md (hunk 1 above)
Comment (francescalb) [reply OntoTrans2ndWorkshop/README.md:21]: I can add a link to the how to. I disagree with a full documentation on how.It is duplication of things that quickly end up being outdated.

File: OntoTrans2ndWorkshop/demo_figures/SimulationParametersExcel.png
Comment (francescalb) [reply OntoTrans2ndWorkshop/demo_figures/SimulationParametersExcel.png:1]: The demo.ipynb was not commited, my mistake.

File: OntoTrans2ndWorkshop/README.md (hunk 1 above)
Comment (jesper-friis) [reply OntoTrans2ndWorkshop/README.md:21]: I was only thinking about a quick summary. This if fine

File: OntoTrans2ndWorkshop/demo_figures/SimulationParametersExcel.png
Comment (jesper-friis) [reply OntoTrans2ndWorkshop/demo_figures/SimulationParametersExcel.png:1]: Very nice. I think there should be a link to demo notebook from the README. It renders nicely in github.

**Review by jesper-friis — APPROVED** [review]

**Pull request merged**